test_app/lib/screens/participant_screen/participant_grid.dart

```diff
@@ -83,7 +83,7 @@ class _ParticipantGridState extends State<ParticipantGrid> {
           var participant = participants[index];
           return ParticipantWidget(
               participant: participant,
-              remoteOptionsFlag: index == 0 ? false : widget.remoteOptionsFlag);
+              remoteOptionsFlag: widget.remoteOptionsFlag);
```

Comment:
The local participant must not have options menu. How are you ensuring that this is not the case.

Reply:
Can you point me to the logic that does that?

Reply:
Line 86. If participant index is 0 remoteOptionFlag is set to false.

Reply:
Sometimes local participant index is not 0 and then we have bug.

Reply:
Thank you very much for the clarification and improving the logic.